packages/components/src/components/file/file.component.ts

```diff
@@ -300,6 +301,56 @@ export default class SynFile extends SynergyElement implements SynergyFormContro
     this.files = files;
   }
 
+  private async handleTransferItems(items: DataTransferItemList | null): Promise<FileList> {
+    if (!items || items.length !== 1) {
+      this.value = '';
+      return new Promise((_resolve, reject) => { reject(new Error('No proper items found')); });
+    }
+
+    const itemsArray = Array.from(items);
```

Comment:
This is more taste, but I would combine those statements (and also get rid of the forced type as TS is able to guess it pretty well):


```suggestion
    const entries = Array
      .from(items)
      .map(item => item.webkitGetAsEntry());
```